xla/service/gpu/gpu_compiler_test.cc

```diff
@@ -106,6 +106,22 @@ class GpuCompilerTest : public HloTestBase {
     return tensorflow::down_cast<GpuCompiler*>(compiler)
         ->RunPostSchedulingPipelines(module, 4 * 1024 * 1024, gpu_device_info);
   }
+
+  // Like GetOptimizedModule, but also runs the backend.
+  absl::StatusOr<std::pair<const HloModule*, std::unique_ptr<OpaqueExecutable>>>
+  GetOptimizedModuleForExecutable(absl::string_view hlo,
```

Comment:
`GetOptimizedModuleAndExecutable`?

Reply:
The idea is that this is the module for the executable (as opposed to what's usually called the OptimizedModule around here, which is not for the executable, but for some intermediate state). Happy to do the rename, but I don't think it's clearer.